crates/core/src/kernel/snapshot/log_data.rs

```diff
@@ -803,16 +810,16 @@ mod datafusion {
             lazy_static::lazy_static! {
                 static ref ROW_COUNTS_EVAL: Arc<dyn ExpressionEvaluator> =  ARROW_HANDLER.get_evaluator(
                     crate::kernel::models::fields::log_schema_ref().clone(),
-                    Expression::column("add.stats_parsed.numRecords"),
+                    Expression::column(vec!["add.stats_parsed.numRecords"].into_iter()),
```

Comment:
I think this may be the cause for the failing DF tests.

```suggestion
                    Expression::column(["add", "stats_parsed", "numRecords"]),
```